src/applications/post-911-gib-status/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Post 9/11 GI Bill Status",
+  "entryFile": "./post-911-gib-status-entry.jsx",
+  "entryName": "post-911-gib-status",
+  "rootUrl": "/education/gi-bill/post-9-11/ch-33-benefit",
+  "production": true
```

Comment:
Should we remove this production flag?

Reply:
Nope, just copied the former manifest. Will remove that too.